compiler/rustc_codegen_cranelift/src/driver/jit.rs

```diff
@@ -127,7 +126,7 @@ pub(super) fn run_jit(tcx: TyCtxt<'_>, backend_config: BackendConfig) -> ! {
             f();
             std::process::exit(0);
         }
-        EntryFnType::Start => {
+        else {
```

Comment:
The formatting is wrong here. Can you please re-format? `./x.py fmt` is not yet supported here, so you need to run `rustfmt +nightly` manually on this file. (Please don't run `cargo fmt` inside `compiler/rustc_codegen_cranelift` to prevent potential conflicts when syncing the subtree.

Reply:
Oh, i didn't know `./x.py fmt` is not yet supported with cg_cranelift... will reformat.

Reply:
Done.